src/routes/accounts/accounts.controller.ts

```diff
@@ -33,4 +36,15 @@ export class AccountsController {
     const auth = request.accessToken;
     return this.accountsService.createAccount({ auth, createAccountDto });
   }
+
+  @Delete(':address')
+  @UseGuards(AuthGuard)
+  @HttpCode(HttpStatus.NO_CONTENT)
+  async deleteAccount(
+    @Param('address', new ValidationPipe(AddressSchema)) address: `0x${string}`,
+    @Req() request: Request,
+  ): Promise<void> {
+    const auth = request.accessToken;
```

Comment:
With the `AuthGuard`, we will already have access to the `AuthPayload` here.

Reply:
I opened [a PR](https://github.com/safe-global/safe-client-gateway/pull/1722) changing this.